### ISSUE TITLE
Do we need Eclipse Commons?

### ISSUE THREAD

**Issue by enikao** [enikao/eclipse-commons#1]
Please leave a comment with +1 or -1 for voting.

Refer to https://www.nikostotz.de/blog/do-we-need-eclipse-commons/ for the whole story.

**Comment by nbrouwers** [comment]
I agree with all of your personal motivations to initiate the Eclipse Commons project. Very good initiative, it fully deserves my support!
